master/getting-started/kubernetes/installation/hosted/calicoctl.yaml

```diff
@@ -12,7 +12,6 @@ metadata:
   name: calicoctl
   namespace: kube-system
 spec:
-  hostNetwork: true
```

Comment:
Any reason behind this change?

Seems fine to me but might cause confusion when folks can't use calicoctl because policy doesn't let them :)

Reply:
No specific reason. Disabled hostNetwork at the suggestion of @ozdanborne . I think he plans to add a policy at some point to prevent users from locking themselves out.

Reply:
Can we leave the `hostNetwork`  change out for now? Would like to get the core stuff in without having to worry about casey's concerns yet.